src/mscorlib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -701,14 +701,15 @@ internal unsafe struct MEMORY_BASIC_INFORMATION {
         internal const String OLE32    = "ole32.dll";
         internal const String OLEAUT32 = "oleaut32.dll";
         internal const String NTDLL    = "ntdll.dll";
+        internal const String ADVAPI32 = "advapi32.dll";
```

Comment:
Actually, it looks like GetUserName is available in OneCore (but not on Windows 7, and not in UWP):
https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.CodeAnalysis/PackageFiles/PinvokeAnalyzer_OneCoreApis.txt